app/models/reminder.js

```diff
@@ -0,0 +1,7 @@
+import DS from 'ember-data'
+
+export default DS.Model.extend({
+  title: DS.attr('string'),
+  body: DS.attr('string'),
+  reminder_id: DS.attr('number')
```

Comment:
Check out the model generated by Mirage under `factories/reminder.js`. Ember auto generates an ID for you, so the id property here is unnecessary.